pom.xml

```diff
@@ -61,6 +66,33 @@
             <classifier>mule-plugin</classifier>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
We need to validate that javax.activation is really needed here. Since I put it in the boot packages as exported it may not be needed.

Reply:
Good catch @pablolagreca, thanks. I just executed the tests, and they worked fine without having this dependency declared

Reply:
BTW @pablolagreca and @aiannucci, what about the scope of the dependencies I'm adding here. Should they have something like a `test` or `provided` one?

Reply:
Pending: review if adding javax.xml.bind as boot package solves the problem without adding these dependencies